Assets/SEE/GraphProviders/MultiGraphProvider.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using SEE.DataModel.DG;
+using SEE.Game.City;
+using SEE.Utils.Config;
+
+namespace SEE.GraphProviders
+{
+    /// <summary>
+    /// MultiGraphProvider is a graph provider for returning a series of graphs for an evolution city.
```

Comment:
Will we only ever use `MultiGraphProvider` for evolution cities or should we keep the description more generic than that?

Reply:
I generalized it.